js/actions/actions.js

```diff
@@ -18,13 +19,28 @@ module.exports = {
     //Determine the action based on the user command. Eventually this shouldn't be an action
     //and the logic should go somewhere else
     var args = text.split(' ');
+    if (args.length === 1) {
```

Comment:
This is clearly a shitty way of handling this issue, I'll come up with something better before merging.